.docker/mongo/databaseInit/initialSetup.js

```diff
@@ -0,0 +1,7 @@
+// Example of javascript script for mongodb
+
+let res = [
+  //db.container.drop()
```

Comment:
You mentioned this working locally... what would the purpose of this script be vs. initialSetup.sh ?

Reply:
Just a leftover from another project. Usually this is helpful to run some js code after spinning up a db.